tgui/packages/tgui/components/LabeledList.tsx

```diff
@@ -43,34 +45,42 @@ const LabeledListItem = (props: LabeledListItemProps) => {
     color,
     textAlign,
     buttons,
+    tooltip,
     content,
     children,
     verticalAlign = 'baseline',
   } = props;
+  const TooltipWrap = (before_wrap) => {
+    return tooltip ? <Tooltip content={tooltip}>{before_wrap}</Tooltip> : before_wrap;
```

Comment:
There is not really a point in integrating this into the component when it can just be a feature of the UI

Reply:
I didn't want to change the component, but it's not possible to make a text hover it without doing this. I should use another component instead of LabeledList if I want to make the tooltips hovering.